### PR TITLE
Insert Seaport trades into Dune v1's nft.trades

### PR DESCRIPTION
**Pull request by hildobby**
I've checked that:

* [X] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [X] the directory tree matches the pattern /sector/blockchain/ e.g. /tokens/ethereum
* [ ] if adding a new model, I added a test
* [X] the filename is unique and ends with .sql
* [X] each file has only one view, table or function defined  
* [X] column names are `lowercase_snake_cased`

### CODE REVIEW

**Comment by masquot** [comment]
Thank you, great work @hildobby. Made a few fixes and got the query running here:
https://dune.com/queries/933568

@sohwak - could you please add your review?

@soispoke - could you please adapt the work of @hildobby for nft_trades_v2_beta?

**Comment by sohwak** [comment]
Thanks a lot for the job you have done, @hildobby. This really helps me.
@masquot, Yes. please continue with this code.

**Comment by soispoke** [comment]
@masquot I just commited the changes to my branch for nft.trades_v2_beta

File: ethereum/nft/trades/insert_seaport.sql [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    , 'Buy' AS category
+    , currency_contract
+    , erc_standard
+    , erc_standard AS erc_types_array
+    , NULL AS erc_values_array
+    , evt_index
+    , 'Trade' AS evt_type
+    , exchange_contract_address
+    , nft_contract_address
+    , nft_contract_address AS nft_contract_addresses_array
```
Comment (masquot) [inline ethereum/nft/trades/insert_seaport.sql:19]: This pattern does not work I'm afraid. Same comment on lines 13, 22, 30 and 32.

NFT trades v1 aggregates multiple transfers in one row. Hence, the usage of an array.

**Review by masquot — CHANGES_REQUESTED** [review]

**Comment by sohwak** [comment]
I'm sorry I didn't care about this.
@masquot , could you change the query to this?
https://dune.com/queries/936718

I matched every column datatype to `nft.trades`.
Although there is a problem that multiple NFT trades are in one row, it will be fixed later.

**Comment by masquot** [comment]
Thank you @sohwak. Applied the updates you suggested in a new commit.

Now there is a runtime error:
> psql:insert_seaport.sql:137: ERROR:  null value in column "nft_contract_address" of relation "trades" violates not-null constraint
DETAIL:  Failing row contains (2022-06-15 00:31:24+00, null, null, erc721, OpenSea, 3, Bundle Trade, 2, Buy, Trade, 48.130971, \x13c10ecb747f71b8aa01304b1fa08383a9163811, \x3b429774e3dfd840abdbb7f5dd3dffc638b107c9, 0.0399, 39900000000000000, ETH, \x0000000000000000000000000000000000000000, \xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2, null, \x00000000006c3852cbef3e08e8df289169ede581, \x000e9a97382b192f9be7520623794f6410289d700568d242eb7454b7693744..., 14964668, {NULL}, {"\\x13c10ecb747f71b8aa01304b1fa08383a9163811"}, {"\\x3b429774e3dfd840abdbb7f5dd3dffc638b107c9"}, {erc721}, {NULL}, null, \x3b429774e3dfd840abdbb7f5dd3dffc638b107c9, \x00000000006c3852cbef3e08e8df289169ede581, null, 0, 1).

**Comment by sohwak** [comment]
That's what I was designed, when there were several different NFTs traded in one transaction, I filled it as `null`

**Comment by sohwak** [comment]
Give me some time to look at the code again.

**Pull request closed without merging**